File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/BannerAdvertVO.kt
package com.sdkwork.app

data class BannerAdvertVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val advertId: String? = null,
    val advertName: String? = null,
    val advertType: String? = null,
    val positionId: String? = null,
    val creativeUrl: String? = null,
    val landingUrl: String? = null,
    val priority: Int? = null,
    val enabled: Boolean? = null,
    val startTime: String? = null,
    val endTime: String? = null,
    val clickCount: Int? = null,
    val impressionCount: Int? = null,
    val bannerSize: String? = null,
    val animationType: String? = null,
    val refreshInterval: Int? = null
)

File: sdkwork-app-sdk-go/types/banner_advert_vo.go
package types

// Banner广告VO
type BannerAdvertVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	AdvertId string `json:"advertId"`
	AdvertName string `json:"advertName"`
	AdvertType string `json:"advertType"`
	PositionId string `json:"positionId"`
	CreativeUrl string `json:"creativeUrl"`
	LandingUrl string `json:"landingUrl"`
	Priority int `json:"priority"`
	Enabled bool `json:"enabled"`
	StartTime string `json:"startTime"`
	EndTime string `json:"endTime"`
	ClickCount int `json:"clickCount"`
	ImpressionCount int `json:"impressionCount"`
	BannerSize string `json:"bannerSize"`
	AnimationType string `json:"animationType"`
	RefreshInterval int `json:"refreshInterval"`
}
